src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Animal {
+  static alive = [];
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+  hide() {
```

Comment:
отступ между методами
```suggestion
  }
  
  hide() {
```